File: 217-contains-duplicate/contains-duplicate.cs
public class Solution {
    public bool ContainsDuplicate(int [] nums){
        HashSet<int> sh = new HashSet<int>();
        foreach (int num in nums){
            if(sh.Contains(num))return true;
            else sh.Add(num);
        } 
        return false;
    }
} 

File: 217-contains-duplicate/contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_set<int> mapping;
        for (int n : nums) {
            if (mapping.find(n) != mapping.end())
                return true;
            mapping.insert(n);
        }
        return false;
    }
};